api/cpp/lib.rs

```diff
@@ -122,6 +122,16 @@ pub unsafe extern "C" fn slint_register_font_from_data(
     )
 }
 
+#[cfg(feature = "std")]
```

Comment:
I thought this was meant to be used without std

Reply:
Oops yes, copy & paste mistake :)